pkg/sync/v8/data/Deployment.apps/openshift-azure-logging/log-analytics-cluster-agent.yaml

```diff
@@ -82,7 +82,7 @@ spec:
           subPath: main.sh
       nodeSelector:
         beta.kubernetes.io/os: linux
-        node-role.kubernetes.io/infra: "true"
+        node-role.kubernetes.io/compute: "true"
```

Comment:
I set this infra as it's a deployment (one per cluster) and the machines on infra are well suited to this (lots of memory).
Also I don't believe it makes a difference as where this runs (as far as it's functionality goes), it is getting inventory.

Reply:
i'm ok with this